MSBuildTasks/InsertSourceLinks.cs

```diff
@@ -134,8 +143,11 @@ private string CreateSrcsrvFile (string pdbFile, string toolPath)
         using (var fileStream = File.CreateText (srcsrvFile))
         {
           var sourceServerFileUrl = string.Format (VcsUrlTemplate, "%var2%");
+          var useWindowsCredentialsPowershellString = UseWindowsCredentials ? "$True" : "$False";
+
           var sourceServerCommandTemplate = string.IsNullOrEmpty (VcsCommandTemplate)
-              ? "powershell.exe -NoProfile -Command \"& {{(New-Object System.Net.WebClient).DownloadFile('{0}','{1}')}}\""
+              ? "powershell.exe -NoProfile -Command \"& " +
```

Comment:
Please move "+" to next line. Operators are always before the code, not at the end of the line.